index.html

```diff
@@ -5,25 +5,25 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Personal Website</title>
-    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="/styles/styles.css" />
   </head>
 
   <body>
-    <header class="page-header">
+    <header class="header">
```

Comment:
You have a lot of `div` elements in your `header`. I think it should be possible to have the same styling without the `div`s. Note that you can give semantic elements such as `header`, `nav` and the headings normal styling and classes like you did with the `div`s. I would encourage you to try to rebuild it!
For example, you can style the `header` with the styles you have. In the end the header should look something like that: 
```suggestion
    <header class="header">
      <h4>Moin, I'm</h4>
      <h1>Mel Grosser</h1>
      <nav>
         <a>....</a>
         <a>...</a>
         <a>...</a>
       </nav>
     </header>
```